py-polars/polars/dataframe/frame.py

```diff
@@ -7249,131 +7249,174 @@ def pivot(
-        "lazy pivot" if you know the unique column values in advance.
+        `pivot` transforms a "long-format" `DataFrame`, where each row represents an
+        observation, into a "wide-format" one, where each element represents an
+        observation.
+
+        `pivot` is only available in eager mode, since the schema of the output
+        `DataFrame` depends on the number of unique values in `columns`. However, if you
+        know these unique values in advance, you can perform a "lazy pivot", as shown in
+        the "Examples" section below.
+
+        Warnings
+        --------
+        `pivot` will introduce `null` values in the output `DataFrame` if not every
+        combination of the values in `index` and `columns` appears in the input
+        `DataFrame`.
 
         Parameters
         ----------
         values
-            Column values to aggregate. Can be multiple columns if the *columns*
-            arguments contains multiple columns as well. If None, all remaining columns
-            will be used.
+            The column(s) whose values will become the values of the output `DataFrame`.
+            If None, all columns not specified in the index and columns arguments will
+            be used.
         index
-            One or multiple keys to group by.
+            The column(s) whose values will become the index of the output `DataFrame`.
         columns
-            Name of the column(s) whose values will be used as the header of the output
-            DataFrame.
+            The column(s) whose values will become the columns of the output
+            `DataFrame`. If these columns are not :class:`String` columns, their values
+            will be coerced to strings, since polars column names must be strings.
         aggregate_function
-            Choose from:
-
-            - None: no aggregation takes place, will raise error if multiple values are in group.
-            - A predefined aggregate function string, one of
-              {'min', 'max', 'first', 'last', 'sum', 'mean', 'median', 'len'}
-            - An expression to do the aggregation.
+            A function to aggregate multiple `values` with the same `index` and
+            `columns` prior to pivoting, equivalent to using :func:`group_by` as a
+            preprocessing step. Choose from:
+
+            - `None`: no aggregation takes place; will raise an error if multiple values
+              are in a group.
+            - One of the strings {`'first'`, `'last'`, `'sum'`, `'max'`,
+              `'min'`, `'mean'`, `'median'`, `'len'`}, to perform predefined
+              types of aggregation.
+            - An expression that performs a custom aggregation, where
+              :func:`polars.element()` represents the multiple `values` in each "group"
+              with the same `index` and `columns`. For example, `aggregate_function='mean'`
+              is short for `aggregate_function=pl.element().mean()`.
```

Comment:
use double-ticks here if it's not a parameter name:
```
              is short for ``aggregate_function=pl.element().mean()``.
```
(same on the line above for
```
``aggregate_function='mean'``
```
)

Reply:
(and similarly in a few other places)

Reply:
actually, this is fine, other places in the docs use this, please ignore this review comment